bin/index.js

```diff
@@ -1,9 +1,11 @@
 #!/usr/bin/env node
 const app = require('../src/server/server');
 const argv = require('yargs').argv;
+const open = require('open');
 
 const PORT = argv.port || 3000;
 
 app.listen(PORT, () => {
   console.log(`Git Explorer now running at localhost:${PORT}`);
+  open(`http://localhost:${PORT}`);
```

Comment:
I think maybe we could make the url a variable?  That way we can reuse it in both console logging and for the call to `open` ?
```javascript
app.listen(PORT, () => {
  const path = `http://localhost:${PORT}`;

  console.log(`Git Explorer now running at ${path}`);
  open(path);
})
```